content/4_Gold/Modular.mdx

```diff
@@ -367,7 +367,120 @@ We can also use the extended Euclidean algorithm. See the module in the
 
 <CPPOnly>
 
-## Templates
+## With Extended Euclidean Algorithm
```

Comment:
this should probably be moved over to that module

Reply:
feel free to add a reference to CRT in the optional section referring to CRT as well

Reply:
Ok I just removed it from there, since it is already covered in extend-euclid module.